charts/vmclarity/templates/apiserver/deployment.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ printf "%s-apiserver" (include "vmclarity.names.fullname" .) }}
+  namespace: {{ .Release.Namespace | quote }}
+  labels: {{- include "vmclarity.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/component: apiserver
+spec:
+  selector:
+    matchLabels: {{- include "vmclarity.labels.matchLabels" . | nindent 6 }}
+      app.kubernetes.io/component: apiserver
+  replicas: {{ .Values.apiserver.replicaCount }}
+  template:
+    metadata:
+      labels: {{- include "vmclarity.labels.standard" . | nindent 8 }}
```

Comment:
I think `matchLabels` should be used here, also for all other deployments?

Reply:
we want all the labels on the pod, not just the ones used for matching

Reply:
do we?

Reply:
IMO yes, when you do `kubectl describe pod vmclarity-apiserver...` I would expect to see the full set of labels including version etc the same as all the other resources not just the name, instance and component ones which are used for matching a pod to a service/deployment.